documentation/staging/content/faq/node-heating.md

```diff
@@ -29,12 +29,37 @@ clusters:
+                    values:
+                    - $(DOMAIN_UID)
               topologyKey: "kubernetes.io/hostname"
 ```
 
-Because the `serverPod` element here is scoped to a cluster, the content of the `affinity` element will be added to the Pod generated for each WebLogic Server instance that is a member of this WebLogic cluster. This inter-pod anti-affinity statement expresses a preference that the scheduler select a Node for the new Pod avoiding, as much as possible, Nodes that already have Pods with the label "weblogic.clusterName" and the name of this cluster. Note that the `weight` is set to `100`, which is the maximum weight, so that this term will outweigh any possible preference for a Node based on availability of container images.
+Because the `serverPod` element here is scoped to a cluster, the content of the `affinity` element will be added to the Pod generated for each WebLogic Server instance that is a member of this WebLogic cluster. This inter-pod anti-affinity statement expresses a preference that the scheduler select a Node for the new Pod avoiding, as much as possible, Nodes that already have Pods with the label "weblogic.clusterName" and the name of this cluster and with the label "weblogic.domainUID" and the UID of this domain. Note that the `weight` is set to `100`, which is the maximum weight, so that this term will outweigh any possible preference for a Node based on availability of container images.
+
+For the server pods that are not part of a WebLogic cluster, the operator adds the following content to the `serverPod` element at the domain scope in the Domain Custom Resource.  The following expresses an anti-affinity, but changes the test to have all WebLogic Server instances in the domain prefer to run on Nodes where there is not already a Pod for a running instance.
+
+```yaml
+clusters:
+- clusterName: cluster-1
+  serverStartState: "RUNNING"
+  serverPod:
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: "weblogic.domainUID"
+                    operator: In
+                    values:
+                    - $(DOMAIN_UID)
+              topologyKey: "kubernetes.io/hostname"
+```
 
-It is possible to express many other scheduling preferences or constraints. The following example similarly expresses an anti-affinity, but changes the test to have all WebLogic Server instances in the domain prefer to run on Nodes where there is not already a Pod for a running instance:
+It is possible to customize the default affinity preferences, and express many other scheduling preferences or constraints. The following example similarly expresses an anti-affinity, but changes the test to have all WebLogic Server instances in the domain prefer to run on Nodes where there is not already a Pod with "security" label with value of "S1".
```

Comment:
preferences, and -> preferences and (no comma needed here)
with "security" label with value of "S1". -> with a `security` label with the value of `S1`.

Reply:
Fixed.